jenkins/freestyle/get-sitespeed-report.sh

```diff
@@ -19,6 +20,7 @@ set -e
 : ${USE_BASIC_AUTH:?"The USE_BASIC_AUTH environment variable must be set."}
 : ${NUMBER_OF_TIMES:?"The NUMBER_OF_TIMES environment variable must be set."}
 : ${CONNECTION:?"The CONNECTION environment variable must be set."}
+: ${SITESPEED_BROWSER:?"The SITESPEED_BROWSER environment variable must be set."}
```

Comment:
Good point. I'm not too worried because it's only jenkins jobs that will be using the script. We probably should refactor in the future. I'll merge this in now tho so I can set the job back to master instead of this branch.